doc/articles/uno-development/building-uno-ui.md

```diff
@@ -15,7 +15,7 @@ This article explains how to build Uno.UI locally, for instance, if you wish to
   - To build the UWP flavor of Uno, you'll need **UWP Development**, install all recent UWP SDKs, starting from 10.0.19041 (or above or equal to `TargetPlatformVersion` line [in this file](https://github.com/unoplatform/uno/blob/master/src/Uno.CrossTargetting.targets))
 - Install (**Tools** / **Android** / **Android SDK manager**) all Android SDKs starting from 7.1 (or the Android versions `TargetFrameworks` [list used here](https://github.com/unoplatform/uno/blob/master/src/Uno.UI.BindingHelper.Android/Uno.UI.BindingHelper.Android.csproj))
 - Run [Uno.Check](https://github.com/unoplatform/uno.check) on your dev machine to setup .NET Android/iOS workloads
-- Install the version of the .NET SDK specified in [global.json](../../../global.json).
+- Install the version of the .NET SDK specified in [global.json](https://github.com/unoplatform/uno/blob/master/global.json).
```

Comment:
We no longer specify a version, it's best to suggest latest I think

```suggestion
- Install the latest .NET SDK.
```

Reply:
What dod you think @jeromelaban?

Reply:
@Youssef1313 @agneszitte IMO this is a good change, maybe we could just add a link to .NET SDK download?

Reply:
Yes I agree if you can add it @morning4coffe-dev please